examples/jsm/renderers/common/Renderer.js

```diff
@@ -875,7 +864,20 @@ class Renderer {
 
 	}
 
-	_renderObjectDirect( renderObject ) {
+	_renderObjectDirect( object, material, scene, camera, lightsNode, passId ) {
+
+		const renderObject = this._objects.get( object, material, scene, camera, lightsNode, this._currentRenderContext, passId );
+
+		//
+
+		this._nodes.updateBefore( renderObject );
+
+		//
+
+		object.modelViewMatrix.multiplyMatrices( camera.matrixWorldInverse, object.matrixWorld );
```

Comment:
Wouldn't these lines be called twice in case of double-pass transparent rendering?

Reply:
> Wouldn't these lines be called twice in case of double-pass transparent rendering?

The `updateBefore()` can made RTT because of some nodes, and this can affect the current `modelViewMatrix`, I'll improve this once I have a better idea, I'm currently prioritizing stability.